templates/includes/sidebar.html

```diff
@@ -1,4 +1,12 @@
 <ul class="sidebar-nav">
+  {% if user.is_authenticated %}
+    <li>User: {{ user.username }}</li>
```

Comment:
according to the task, you should have implemented "If a driver clicks on a username - the corresponding driver information page should open".